tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -675,6 +675,90 @@ def try_get_features():
     assert online_features_actual.to_dict() == online_features_expected
 
 
+@pytest.mark.timeout(600)
+@pytest.mark.run(order=18)
+def test_basic_retrieve_feature_row_missing_fields(client, cust_trans_df):
+    # reset the project to default
+    client.set_project()
+
+    feature_refs = ["daily_transactions", "total_transactions", "null_values"]
+    # update cust_trans_fs with one additional feature.
+    # feature rows ingested before the feature set update will be missing a field.
+    old_cust_trans_fs = client.get_feature_set(name="customer_transactions")
```

Comment:
Our tests are becoming more and more brittle over time if we have these expectations of feature sets being registered already. One change upstream breaks multiple tests. Over time I want to make them a bit more atomic, which would also allow for higher parallelism.

Possible to add your own `apply` here for the first feature set?

Reply:
Noted. Added.